pkg/agent/main.go

```diff
@@ -348,3 +366,38 @@ func certinfo(cert *x509.Certificate) {
 	logrus.Infof("SignatureAlgorithm: %+v", cert.SignatureAlgorithm)
 	logrus.Infof("PublicKeyAlgorithm: %+v", cert.PublicKeyAlgorithm)
 }
+
+func reconcileKubelet(ctx context.Context) (bool, error) {
+	if os.Getenv("CATTLE_K8S_MANAGED") == "true" {
+		return true, nil
+	}
+
+	nodeName := os.Getenv("CATTLE_NODE_NAME")
+	logrus.Infof("node %v is not registered, try restarting kubelet now", nodeName)
```

Comment:
"try restarting" is a little odd. can you drop try and say "restarting kubelet now"